core/src/main/java/org/infinispan/configuration/cache/AbstractStoreConfigurationBuilder.java

```diff
@@ -55,10 +54,12 @@ public AbstractStoreConfigurationBuilder(PersistenceConfigurationBuilder builder
 
    /**
     * {@inheritDoc}
+    *
+    * @deprecated Deprecated since 14.0. There is no replacement. First non shared store is picked instead.
     */
+   @Deprecated
    @Override
    public S fetchPersistentState(boolean b) {
```

Comment:
I would deprecate the method in `StoreConfigurationChildBuilder`.

Reply:
Sure.